src/app/components/fireside/avatar/AppFiresideAvatar.vue

```diff
@@ -269,12 +279,25 @@ async function ejectFireside(community: FiresideCommunity) {
 							@{{ fireside.user.username }}
 						</div>
 
-						<div v-if="community" class="-tooltip-row -tooltip-community">
+						<div v-if="community || realm" class="-tooltip-row -tooltip-muted">
 							<AppTranslate>in</AppTranslate>
-							<div class="-tooltip-img">
-								<AppCommunityThumbnailImg :community="community" />
-							</div>
-							{{ community.name }}
+							<template v-if="community">
+								<div class="-tooltip-img">
+									<AppCommunityThumbnailImg :community="community" />
+								</div>
+								{{ community.name }}
+
+								<template v-if="realm">
+									<AppTranslate>and</AppTranslate>
+								</template>
+							</template>
+
+							<template v-if="realm">
+								<div class="-tooltip-img-realm">
+									<AppRealmThumbnail :realm="realm" not-rounded />
+								</div>
+								{{ realm.name }}
+							</template>
```

Comment:
ah, i didnt actually touch these, are we fine doing these random refactors in PRs? I've been trying to avoid these to make the PR smaller and more readable.

Reply:
is there a better way than this mess?
```
<template v-if="i.isFeatured">
	{{ $gettext(`Unfeature fireside`) }}
</template>
<template v-else>
	{{ $gettext(`Feature fireside`) }}
</template>
```
If i remember correctly we must use constant expressions within our $gettexts but this feels horribly wordy

Reply:
this is not very readable to me ```{{ i.isFeatured ? $gettext(`Unfeature fireside`) : $gettext(`Feature fireside`) }}```

Reply:
I think you would get used to the ternary way honestly. it's pretty readable to me. also, we talked in chat but you would do $gettextInterpolate($ngettext(...), {})